test/kubernetes_unittest.cc

```diff
@@ -1951,4 +1966,52 @@ TEST_F(KubernetesTestFakeServerThreeWatchRetriesNodeLevelMetadata,
   EXPECT_TRUE(server->TerminateAllStreams(time::seconds(3)));
 }
 
+namespace {
+class MockSleeper : public Sleeper {
+ public:
+  MOCK_METHOD1(SleepFor, void(double seconds));
+};
+
+// Polls health_checker until it has at least one unhealthy component.
+// Returns false if newer timestamp not found after 3 seconds (polling
```

Comment:
if using `this_thread::sleep_for()`, the comments should be at least 3 seconds (polling every 100 millis at least).
ref: https://en.cppreference.com/w/cpp/thread/sleep_for

Reply:
Done.